src/Resources/translations/SonataMediaBundle.de.xliff

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
This change is a problem to our validator @VincentLanglet

Reply:
dunno why `'` is used here and. `"` everywhere else.
dunno if we can configure the validator for this...